RetrospectiveExtension.Backend/Startup.cs

```diff
@@ -87,10 +89,15 @@ public void ConfigureServices(IServiceCollection services)
 
       services.AddSignalR().AddAzureSignalR(Configuration.GetValue<string>("Azure:SignalR:ConnectionString"));
       services.Configure<AppInsightsSettings>(Configuration.GetSection("ApplicationInsights"));
+
+      services.AddLogging(builder => {
+           builder.AddFilter<ApplicationInsightsLoggerProvider>("",LogLevel.Information);
```

Comment:
Should the level come from the app settings?

Reply:
great point After some expirementation I found that simply adding ` services.AddApplicationInsightsTelemetry();` would automatically add a logger and its level can be configured from `appsettings.json`. Hence I removed this line and put App Inisghts log level settings in the `appsettings`. This [resource](https://docs.microsoft.com/en-us/azure/azure-monitor/app/ilogger#logging-level) helped me a lot